asv_bench/benchmarks/tslibs/period.py

```diff
@@ -104,3 +112,19 @@ def setup(self, size, freq):
 
     def time_periodarray_to_dt64arr(self, size, freq):
         periodarr_to_dt64arr(self.i8values, freq)
+
+
+class TimeDT64ArrToPeriodArr:
+    params = [
+        _sizes,
+        _freq_ints,
+        _tzs,
+    ]
+    param_names = ["size", "freq", "tz"]
+
+    def setup(self, size, freq, tz):
+        arr = np.arange(10, dtype="i8").repeat(size // 10)
+        self.i8values = arr
+
+    def time_dt64arr_to_periodarr(self, size, freq, tz):
+        dt64arr_to_periodarr(self.i8values, freq, tz)
```

Comment:
Is it possible to time the public function/method that in the end uses this instead?

Reply:
its possible, yes.  but ATM im aiming for thorough coverage of tslibs in isolation, and PRs that dont touch tslibs can skip any of their asvs

Reply:
we likely *already* have coverage at a higher level, these fine-grained asv's are useful.